crates/plugins/src/store.rs

```diff
@@ -24,6 +24,17 @@ impl PluginStore {
     }
 
     pub fn try_default() -> Result<Self> {
+        if let Ok(brew_prefix) = std::env::var("HOMEBREW_PREFIX") {
```

Comment:
This may be premature, but is it worth pulling the "package manager stanza" out into a separate function?  It almost overwhelms the main flow here, and doesn't really state intent.  Maybe even pull out the whole "figure out the right directory" logic to make early returns easier e.g.

```
fn try_default() {
  Self::new(default_dir()?))
}

fn default_dir() {
  if let Ok(test_dir) = std::env::var(...) {
    return test_dir;
  }
  if let Some(override) = package_manager_plugins_dir() {
    return override;
  }
  // XDG logic
}
```

I'm thinking out loud here - feel free to tell me this is a bad idea!

Reply:
I like this. It's nice and clean